src/Makefile.test.include

```diff
@@ -60,6 +60,7 @@ BITCOIN_TESTS =\
   test/merkle_tests.cpp \
   test/miner_tests.cpp \
   test/multisig_tests.cpp \
+  test/net_tests.cpp \
```

Comment:
I am testing CAddrDB, which is called from and lives in net.h and net.cpp not CAddrMan.